packages/all/project.json

```diff
@@ -97,7 +100,7 @@
         "parallel": false,
         "commands": [
           "nx run ag-charts-website-gallery_simple-bar_main.ts:generate-thumbnail",
-          "node {projectRoot}/watch.js & nx run ag-charts-website:dev"
+          "node {projectRoot}/../../external/ag-website-shared/scripts/watch/watch.js charts & nx run ag-charts-website:dev"
```

Comment:
Nit: Maybe `{workspareRoot}/external/ag-website-shared/scripts/watch/watch.js` would be slightly more robust, rather than using `../..`?

Reply:
I did try this before, and I get this error

```
NX  The {workspaceRoot} token is only valid at the beginning of an option. (all:dev.commands)
```

Reply:
Removing `{projectRoot}/../../` fixes it, as the `workspaceRoot` is implied.

As per https://github.com/nrwl/nx/issues/14436#issuecomment-1778089590